README.md

```diff
@@ -21,7 +21,7 @@ A library and CLI app for rendering project templates.
 
 ## Installation
 
-1. Install Python 3.6.1 or newer (3.8 or newer if you're on Windows).
+1. Install Python 3.7 or newer (3.8 or newer if you're on Windows).
```

Comment:
This deserves something in the changelog

Reply:
Oh yes, the changelog

Reply:
Should it go in Deprecated, Removed, Changed?

Reply:
That's removed, right?